app/components/join-section.hbs

```diff
@@ -40,14 +40,13 @@
     
     {{!-- NOTE: This has the new code on feature flag off --}}
     {{!-- If something goes wrong, we can enable a flag to restore previous behaviour --}}
-    {{!-- TODO: Clean up by Jan, 2025 --}}
+    {{!-- TODO: Clean up by Feb, 2025 --}}
+    {{#if this.isDevMode}}
     <p data-test-join-later-text class='join__later-text'>
-      {{#if this.isDevMode}}
       We're currently not accepting new applications. Please come back later.
-      {{else}}
-      <NoticeForJoining />
-      {{/if}}
     </p>
-
+    {{else}}
+    <NoticeForJoining data-test-join-later-text class="join__later-text"/>
```

Comment:
I have noticed that the `<p>` and the component `NoticeForJoining` has same data test id's. 

we can keep the ID's different to avoid ambiguity and that will make it easier to target the correct element during testing

Reply:
I would like to differ. The final element that has to be tested remains the same, whether FF is on or off :)